src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
Probably fixed in the extension itself as I cannot reproduce it anymore. Yes @jmalloc the warning I was talking was additional and not to related to the actual error generated inside a callback, this made things very lousy. I'm absolutely :+1: to remove the @- suppression, it's a stupid hack and I hated to have it back then.